docs/source/tutorials/memory_optimisations.rst

```diff
@@ -0,0 +1,243 @@
+.. _memory_optimisation_overview_label:
+
+============================
+Memory Optimization Overview
+============================
+
+**Author**: `Salman Mohammadi <https://github.com/SalmanMohammadi>`_
```

Comment:
does this look weird?

Reply:
We haven't done this in other tutorials, but I've seen pytorch tutorials that do call out authors (i.e., https://pytorch.org/tutorials/recipes/distributed_checkpoint_recipe.html). So no strong opinion here